DependencyInjection/DoctrineMongoDBExtension.php

```diff
@@ -508,8 +505,7 @@ protected function loadDocumentManagerBundlesMappingInformation(array $documentM
         $odmConfigDef->addMethodCall('setDocumentNamespaces', [$this->aliasMap]);
     }
 
-    /** @param string $name */
-    protected function getObjectManagerElementName($name): string
+    protected function getObjectManagerElementName(string $name): string
```

Comment:
[`AbstractDoctrineExtension::getObjectManagerElementName()` already has `string` in 5.4](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Bridge/Doctrine/DependencyInjection/AbstractDoctrineExtension.php#L453)